src/ensembl/src/shared/components/help-article/VideoArticle.tsx

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
Do we need to keep this file?

Reply:
Eventually yes, but in this PR probably not. I'll remove it from this PR and add it back later.

Reply:
Removed.